docs/devnotes.md

```diff
@@ -18,3 +18,55 @@ git update-index --assume-unchanged ios/ConfigValues.h
 
 When changing them turn the flag off, update, commit, turn flag back on.
 More details: https://git-scm.com/docs/git-update-index#_using_assume_unchanged_bit
+
+## Build fails on missing config files
+
+After following the steps in the [quickstart guide](./quickstart.md) your builds depend on the generated config files. When these files are missing, you may see build errors like:
+
+> yaml file at path /X/Y/node_modules/react-native-ultimate-config/android/rnuc.yaml does not exist
+
+Or
+
+> /X/Y/node_modules/react-native-ultimate-config/ios/ConfigValues.h:1:2: error: "invoke bin.js with env file before compiling native project"
+
+When this happens, all you need to do is trigger the script that generates the required config files:
+
+```bash
+yarn rnuc [PATH TO YOUR ENV FILE]
+```
+
+☝️ Note that the generated config files are not persistent in your codebase (and they should not be). This can lead to builds failing when:
+
+- Cleaning your local working directory (e.g. when re-installing node-modules).
+- Initiating the project on a new machine.
+- Building the project on a CI server.
+
+To prevent this, simply add the `yarn rnuc [PATH TO ENV FILE]` command in your workflow before building.
+
+### 🧹 Example: cleaning your local working directory
+
+```json
+// package.json
+
+{
+  "scripts": {
+    "clean": "rm -rf node_modules && yarn && yarn rnuc ./env.yaml"
```

Comment:
this script makes an assumption about name and format of the env file - not sure if this should be hardcoded here, what do you think?

Reply:
True, I explicitly did not use that in the leading documentation. My assumption was that in an example it could help clarify the syntax of the command without implying this should be the name or format used. If you prefer I can change it?